tests/test_jobs/test_create_terms_schema.py

```diff
@@ -111,15 +111,15 @@ def test_post_summer_grace_period(self, mock_datetime, app, term_definitions):
     def test_early_fall(self, mock_datetime, app, term_definitions):
         mock_datetime.now.return_value = datetime(year=2018, month=8, day=15, hour=12, minute=21)
         terms = self.refresh_term_index(app)
-        assert terms['current_term_name'] == 'Fall 2018'
+        # assert terms['current_term_name'] == 'Fall 2018'
         assert terms['future_term_name'] == 'Fall 2018'
 
     @mock.patch('nessie.jobs.create_terms_schema.datetime', autospec=True)
     def test_mid_fall(self, mock_datetime, app, term_definitions):
         mock_datetime.now.return_value = datetime(year=2018, month=10, day=13, hour=12, minute=21)
         terms = self.refresh_term_index(app)
         assert terms['current_term_name'] == 'Fall 2018'
-        assert terms['future_term_name'] == 'Spring 2019'
+        # assert terms['future_term_name'] == 'Spring 2019'
 
```

Comment:
@sandeep-jay Were these test lines commented out as a temporary measure?

Reply:
timezone change weirdness. Reinstated the assertions and Travis seems happy.